### PR TITLE
Fixing validations

### PR DESCRIPTION
**Pull request by antopalidi**

### CODE REVIEW

File: spec/forms/decidim/participatory_documents/admin/document_form_spec.rb [hunk 1]
```diff
@@ -59,6 +61,22 @@ module Decidim::ParticipatoryDocuments::Admin
           expect(subject.box_opacity).to be_nil
         end
       end
+
+      context "when the document has no file" do
```
Comment (microstudi) [inline spec/forms/decidim/participatory_documents/admin/document_form_spec.rb:65]: Are we missing a the case of uploading an invalid file?

File: spec/forms/decidim/participatory_documents/admin/document_form_spec.rb [hunk 2]
```diff
@@ -59,6 +61,22 @@ module Decidim::ParticipatoryDocuments::Admin
...
+        it "is invalid" do
+          expect(subject).to be_valid
+        end
+      end
+
+      context "when the document has a file with an invalid extension" do
+        let(:file) { upload_test_file(Decidim::Dev.test_file("dummy-dummies-example.json", "application/pdf")) }
+
+        it "is valid" do
+          expect(subject).to be_valid
```
Comment (microstudi) [inline spec/forms/decidim/participatory_documents/admin/document_form_spec.rb:77]: shouldnt this case be invalid?

**Review by microstudi — COMMENTED** [review]
Looks good, can you check a couple of comments?

File: spec/forms/decidim/participatory_documents/admin/document_form_spec.rb (hunk 1 above)
Comment (antopalidi) [reply spec/forms/decidim/participatory_documents/admin/document_form_spec.rb:65]: this is done

**Review by microstudi — APPROVED** [review]

**Pull request merged**